src/common/include/com/amazonaws/kinesis/video/common/CommonDefs.h

```diff
@@ -364,6 +364,8 @@ typedef CID*                PCID;
 #define MAX_INT64           ((INT64) 0x7fffffffffffffff)
 #define MIN_INT64           ((INT64) 0x8000000000000000)
 

```

Comment:
Don’t really need this. You could use MAX_UINT32 instead

Reply:
Removed